config/filter.d/ignorecommands/apache-fakegooglebot.sh

```diff
@@ -0,0 +1,21 @@
+#!/bin/sh
+# Based on: https://isc.sans.edu/forums/diary/When+Google+isnt+Google/15968/
```

Comment:
It is a forum post with no copyright/license terms I could see. I removed the parsing of the log file parts since fail2ban is providing the IP as an argument. What is left is basically just comparing results of `host` and parsing the DNS answer.